tools/buildsteps/windows/arm-uwp/BuildSetup.bat

```diff
@@ -9,7 +9,8 @@ IF ERRORLEVEL 1 (
   EXIT /B 1
 )
 
-SET cmakeGenerator=Visual Studio %vsver% ARM
+SET cmakeGenerator=Visual Studio %vsver%
+SET cmakeArch=-A ARM -T host=x64
```

Comment:
The changes are also good for me.
Only not complete sure about the value set.

Maybe to have in "BuildSetup.bat " by cmake.exe direct the `-T host=x64`
or here:
```
SET cmakeGenerator=Visual Studio %vsver%
SET cmakeArch=ARM
SET cmakeHost=x64
```

And then on other:
```
cmake.exe -T host=%cmakeHost% -G "%cmakeGenerator%" -A %cmakeArch% %cmakeProps% %base_dir%
```

Reply:
Seem right. It's a little more elaborated.

Since `host=x64` is common for all, I leave it in the main file.